docs/docs/developer_guide/documentation.md

```diff
@@ -0,0 +1,68 @@
+# Mechaphlowers Documentation
+
+## Building documentation 
+
+### Serving documnentation
+
+From inside docs folder : 
+```{bash}
+cd ./docs
+mkdocs serve -a 0.0.0.0:8001
```

Comment:
Suggestion: add a poetry command to locally serve the documentation?

Reply:
ok